variables.tf

```diff
@@ -249,6 +249,21 @@ variable "vpc_config" {
   })
 }
 
+variable "logging_config" {
+  description = "The function's Amazon CloudWatch Logs configuration settings."
+  default     = null
+  type = object({
+    log_format            = string
+    application_log_level = optional(string, null)
+    log_group             = optional(string, null)
+    system_log_level      = optional(string, null)
+  })
+  validation {
+    condition     = var.logging_config == null || (var.logging_config.log_format == "JSON" || var.logging_config.log_format == "Text")
+    error_message = "log_format must be either 'JSON' or 'Text'"
+  }
```

Comment:
the`log_format` is validated at plan time by terraform anyways, so we could remove the validation here and keep the terraform compatibility with `1.5.7`